addons/ai/CfgVehicles.hpp

```diff
@@ -0,0 +1,12 @@
+class CfgVehicles {
+    class CUP_O_RU_Soldier;
+        class CUP_O_RU_JTAC_Flora: CUP_O_RU_Soldier {
```

Comment:
Change indentation to be inline. Confusing otherwise in terms of closing brackets.